hokusai/commands/review_app.py

```diff
@@ -13,11 +13,39 @@
   yaml_content_with_header
 )
 from hokusai.lib.config import HOKUSAI_CONFIG_DIR, HOKUSAI_TMP_DIR, config
+from hokusai.services.configmap import ConfigMap
 from hokusai.services.kubectl import Kubectl
 from hokusai.services.namespace import Namespace
+from hokusai.services.service_account import ServiceAccount
 from hokusai.services.yaml_spec import YamlSpec
 
 
+def copy_configmap(name, destination_namespace):
+  ''' copy configmap from default namespace to destination namespace '''
+  source_configmap = ConfigMap('staging', name=name)
+  destination_configmap = ConfigMap(
+    'staging',
+    name=name,
+    namespace=destination_namespace
+  )
+  source_configmap.load()
+  destination_configmap.struct['data'] = source_configmap.struct['data']
+  destination_configmap.save()
+
+def copy_sa(name, destination_namespace):
```

Comment:
Non-blocking, but perhaps its better to be more declarative here and use `copy_service_account`?

Reply:
sure